docs/identity.md

```diff
@@ -0,0 +1,412 @@
+- [Identity in OSM](#identity-in-osm)
+- [How Identity is represented in OSM](#how-identity-is-represented-in-osm)
+  - [The role of Certificates](#the-role-of-certificates)
+- [Establishing a Pods Identity](#establishing-a-pods-identity)
+  - [Pod Admission](#pod-admission)
+    - [Envoy Sidecar](#envoy-sidecar)
+      - [Confirming Pod Identity when requesting configuration from xDS](#confirming-pod-identity-when-requesting-configuration-from-xds)
+    - [Service to Service communication certificates](#service-to-service-communication-certificates)
+- [Identity in Action](#identity-in-action)
+  - [SMI Traffic Policy](#smi-traffic-policy)
+  - [Permissive Mode](#permissive-mode)
+
+## Identity in OSM
+
+To provide features like mTLS, traffic policies and traffic splits, a service mesh assigns each pod an Identity.
+
+This happens in two steps.  First OSM will need to establish an Identity for
+each pod. Second when the pods are communicating,  `Pod A` will need to confirm `Pod B`'s identity and vice versa. This document will go over how this is achieved in OSM.
+
+## How Identity is represented in OSM
+
+Internally, the [Identity](https://github.com/openservicemesh/osm/tree/main/pkg/identity) of an application is represented as a string. It is the unique string for an
+application which represents enough information to be uniquely identified within a given environment. The string
+format could have multiple representations depending on the environment the
+application is running in such as a VM, Kubernetes, or cloud service.  Currently
+the only environment that is represented in OSM is Kubernetes.
+
+An example of an Identity in Kubernetes for an application is the following. It uses the Service Account assigned to the Pod and the namespace the pod is running in:
+
+```
+<ServiceAccount>.<Namespace>
+```
+
+The application's Identity is also bound to a Trust Domain. A Trust Domain is a logical
+boundary in which the application can be trusted.  By default, OSM's Trust Domain is
+`cluster.local`. This Trust Domain is encoded in the certificate signing
+requests (CSR) so it can be used during mTLS setup. This Trust Domain is configurable in OSM because some Cluster Authorities (CA's) may have policies
```

Comment:
```suggestion
request (CSR) so it can be used during mTLS setup. The Trust Domain is configurable in OSM because some Cluster Authorities (CA's) may have policies
```